modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/base/AssetListEntryLocalServiceBaseImpl.java

```diff
@@ -0,0 +1,716 @@
+	 * </p>
+	 *
+	 * @param start the lower bound of the range of asset list entries
+	 * @param end the upper bound of the range of asset list entries (not inclusive)
+	 * @return the range of asset list entries
+	 */
+	@Override
+	public List<AssetListEntry> getAssetListEntries(int start, int end) {
+		return assetListEntryPersistence.findAll(start, end);
+	}
+
+	/**
+	 * Returns the number of asset list entries.
+	 *
+	 * @return the number of asset list entries
+	 */
+	@Override
+	public int getAssetListEntriesCount() {
+		return assetListEntryPersistence.countAll();
+	}
+
+	/**
+	 * Updates the asset list entry in the database or adds it if it does not yet exist. Also notifies the appropriate model listeners.
+	 *
+	 * @param assetListEntry the asset list entry
+	 * @return the asset list entry that was updated
+	 */
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public AssetListEntry updateAssetListEntry(AssetListEntry assetListEntry) {
+		return assetListEntryPersistence.update(assetListEntry);
+	}
+
+	@Override
+	public AssetListEntryLocalization fetchAssetListEntryLocalization(
+		long assetListEntryId, String languageId) {
+		return assetListEntryLocalizationPersistence.fetchByAssetListEntryId_LanguageId(assetListEntryId,
+			languageId);
+	}
+
+	@Override
+	public AssetListEntryLocalization getAssetListEntryLocalization(
+		long assetListEntryId, String languageId) throws PortalException {
+		return assetListEntryLocalizationPersistence.findByAssetListEntryId_LanguageId(assetListEntryId,
+			languageId);
+	}
+
+	@Override
+	public List<AssetListEntryLocalization> getAssetListEntryLocalizations(
+		long assetListEntryId) {
+		return assetListEntryLocalizationPersistence.findByAssetListEntryId(assetListEntryId);
+	}
+
+	@Override
+	public AssetListEntryLocalization updateAssetListEntryLocalization(
+		AssetListEntry assetListEntry, String languageId, String title,
+		String description) throws PortalException {
+		assetListEntry = assetListEntryPersistence.findByPrimaryKey(assetListEntry.getPrimaryKey());
+
+		AssetListEntryLocalization assetListEntryLocalization = assetListEntryLocalizationPersistence.fetchByAssetListEntryId_LanguageId(assetListEntry.getAssetListEntryId(),
+				languageId);
+
+		return _updateAssetListEntryLocalization(assetListEntry,
+			assetListEntryLocalization, languageId, title, description);
+	}
+
+	@Override
```

Comment:
Is there anything different in the behavior of this method then what you guys are doing?  I know the it's a Map<String, String> but this is a new service, you should be able to create the map and pass it down to the service.